omnibus/Gemfile

```diff
@@ -1,13 +1,14 @@
 source 'https://rubygems.org'
 
 gem 'rake'
-
+# chef-cli pulls chef >= 15.0 which is 16.1.16 at the time of failure.
+# https://github.com/chef/chef/commit/003fbc132935961e93667f11c3f45ce4914b83ac
+gem 'chef', '=15.12.22'
 gem 'chefspec'
 # chef-cli >=3.0.4 requires Ruby version >= 2.7 which chef-server isn't ready for
 gem 'chef-cli', '=3.0.1'
-
 gem 'berkshelf'
-gem 'bundler', '>1.10'
+gem 'bundler', '=1.17.2'
```

Comment:
Note for other reviewers, our current plan is to try so sort out the bundler version shenanigans once oc-id is updated, as that has been the main pain-point in attempting to upgrade to bundler 2 in the past.